device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D100C12S2/pg_profile_lookup.ini

```diff
@@ -0,0 +1,17 @@
+#  PG lossless profiles.
```

Comment:
We don't need to add a new file. We can just point it to the one in `Mellanox-SN3800-D112C8`